client/services/networks.go

```diff
@@ -3,28 +3,31 @@ package services
 import (
 	"context"
 
-	"github.com/Azure/azure-sdk-for-go/services/network/mgmt/2020-08-01/network"
+	"github.com/Azure/azure-sdk-for-go/services/network/mgmt/2020-11-01/network"
```

Comment:
when you upgrade the version of the service all other resource structs might change, I would double check all existing structs are generated the same with newer version.